docs/source/examples/mitiq-paper/mitiq-paper-codeblocks.md

```diff
@@ -167,35 +167,29 @@ print("Original circuit:", circ, sep="\n")
 +++ {"id": "_5Q3P1DJfcar"}
 
 **Codeblock 9**
+<!-- #endregion -->
 
-```{code-cell} ipython3
----
-colab:
-  base_uri: https://localhost:8080/
-id: tSWVDcJIeuck
-outputId: e9d0c874-8814-4da0-c54b-a6881d3e614a
----
-folded = zne.scaling.fold_gates_from_left(
-    circ, scale_factor=2
-)
-print("Folded circuit:", folded, sep="\n")
+> Note: The method `zne.scaling.fold_gates_from_left` has been deprecated as of v0.36.0
+
+```python colab={"base_uri": "https://localhost:8080/"} id="tSWVDcJIeuck" outputId="e9d0c874-8814-4da0-c54b-a6881d3e614a"
+# folded = zne.scaling.fold_gates_from_left(
+#     circ, scale_factor=2
+# )
+# print("Folded circuit:", folded, sep="\n")
```

Comment:
I think if we are using the `python` directive, then we don't need to comment out the code. Sphinx should just highlight the code without executing it (needs testing!).

```suggestion
```python
folded = zne.scaling.fold_gates_from_left(
    circ, scale_factor=2
)
print("Folded circuit:", folded, sep="\n")
```